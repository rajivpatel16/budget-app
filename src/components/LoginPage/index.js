import React from 'react';
import { connect } from 'react-redux';
import { startLogin } from '../../actions/auth';

const LoginPage = ({startLogin}) => {
    return (
        <div className="box-layout">
        <div className="box-layout__box">
        <h1 className="layout__title">Expenesify App</h1>
        <p>It's time to get your expenese under control.</p>
        <button onClick={startLogin}>Login with Google</button>
        </div>
        </div>
    )
}
const mapDispatchToProps = (dispatch) => ({
    startLogin: () => dispatch(startLogin())
});

export default connect(undefined,mapDispatchToProps) (LoginPage);