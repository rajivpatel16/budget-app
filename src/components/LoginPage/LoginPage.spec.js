import React from 'react';
import { shallow } from 'enzyme';
import  LoginPage  from '../LoginPage';

test('should correclty render login page',()=> {
    const wrapper = shallow(<LoginPage />)
    expect(wrapper).toMatchSnapshot();
})
