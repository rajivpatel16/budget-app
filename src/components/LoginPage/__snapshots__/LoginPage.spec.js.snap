// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should correclty render login page 1`] = `
<div>
  <button>
    Login
  </button>
</div>
`;
